config/rails.yml

```diff
@@ -40,3 +40,29 @@ Rails/SkipsModelValidations:
 # relationships are just # many-to-many, and that's it.
 Rails/HasAndBelongsToMany:
   Enabled: false
+
+# While using 'inverse_of' can reduce DB queries, we have
+# not found this to be a problem in practice. The advantage
+# of turning this on would be that we make the inverse
+# behaviour explicit everywhere ActiveRecord can't apply it
+# automatically, but this is rarely a surprise for developers.
+# We also don't want to add 'inverse_of: false' everywhere;
+# at the time of writing, there is no auto-correct for this.
+Rails/InverseOf:
+  Enabled: false
+
+# This is incompatible with the more robust use of foreign
+# key constraints, which provide the same behaviour.
+#
+# Example: https://github.com/alphagov/content-publisher/blob/f26d9b551842fdf2084159b5b7f1bb078da56936/db/schema.rb#L396
+Rails/HasManyOrHasOneDependent:
+  Enabled: false
+
+# At the time of writing, we have disabled this Cop in a lot
+# of repos. The common problem seems to be that the steps to
+# fix any issues are unclear. It's unclear if this Cop has
+# ever prevented a vulnerability. Disabling this Cop expicitly
+# avoids a false sense of security; there is no intent to
+# remove the many overrides we have for it.
+Rails/OutputSafety:
+  Enabled: false
```

Comment:
I'd be inclined to agree about disabling this. Enabling it feels like it'd cause some serious pain due to the proliferation of `html_safe` across repos.

I had a look at the cop documentation and the techniques they suggest like content_tag or safe_join don't seem applicable to our common approach of pulling HTML out of content items.

Like a few of the rubocop-rails cop this feels potentially a little too opinionated compared to common practices.

I'd suggest updating the comment though, as it took me a search of the cop to understand it was about `html_safe` since the name is quite ambiguous. I also think some of the phrasing in this comment feels a bit like "we know it's a problem but we don't know how to fix it" whereas instead we trust that developers use html_safe responsibly.

Reply:
@kevindew I've had another go at the comment - could you take a look?

Reply:
Sounds good 👍